src/connectors/indie88.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// Indie88 has the track and artist flipped on their main web player. The values are intentionally flipped in this Connector
+
+Connector.playerSelector = 'div.cobrp-player-main, div.mediaplayer-mainbody';
+
+Connector.artistSelector = 'div.cobrp-player-main div.cobrp-player-artist, div.mediaplayer-mainbody .mediaplayer-snp p#track-info-title span';
+
+Connector.trackSelector = 'div.cobrp-player-main .cobrp-player-song, div.mediaplayer-mainbody .mediaplayer-snp p#track-info-artist span';
+
+Connector.onReady = Connector.onStateChanged;
```

Comment:
Does the connector works fine w/o this line? This is a workaround for some websites, and it should not be used everywhere.

Reply:
I just tested and it does seem to work without this line. Should I push that update to the PR?